src/main/scala/beam/agentsim/agents/ridehail/VehicleCentricMatchingForRideHail.scala

```diff
@@ -150,15 +150,15 @@ class VehicleCentricMatchingForRideHail(
   }
 
   private def greedyAssignment(trips: List[AssignmentKey]): List[AssignmentKey] = {
-    val Rok = mutable.ListBuffer.empty[CustomerRequest]
-    val Vok = mutable.ListBuffer.empty[VehicleAndSchedule]
+    val Rok = mutable.HashSet.empty[CustomerRequest]
```

Comment:
@haitamlaarabi please, take a look to these changes. It should not break anything, but still.